src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -150,6 +163,35 @@ public static void main(String[] args) {
     System.exit(0);
   }
 
+  List<String> parseDepgraphRoots() {
+    List<String> result = new ArrayList<>();
+    for (String file : opts.depgraphFiles) {
+      try (FileReader reader = new FileReader(new File(file))) {
+        List<List> list = new Gson().fromJson(reader, new TypeToken<List<List>>(){}.getType());
```

Comment:
Oh, and for now I've created the Depgraph class so users see this as a Java structure, meaning that refactoring will be local to the file